src/datasets/datasets.service.ts

```diff
@@ -355,6 +419,15 @@ export class DatasetsService {
     }
   }
 
+  async getDatasetsWithoutId() {
+    try {
+      const datasets = this.datasetModel.find({}, { _id: 0 }).exec();
+      return datasets;
+    } catch (error) {
+      throw new NotFoundException();
+    }
+  }
+
```

Comment:
What does this function do?

Reply:
Could would you please insert a comment that this function is used only when a manual sync is triggered.